common-npm-packages/azure-arm-rest-v2/AzureServiceClientBase.ts

```diff
@@ -82,7 +82,8 @@ export class AzureServiceClientBase {
 
         // process query paramerters
         queryParameters = queryParameters || [];
-        queryParameters.push('api-version=' + encodeURIComponent(apiVersion || this.apiVersion));
+        const currentApiVersion = apiVersion || this.apiVersion;
+        if (currentApiVersion) { queryParameters.push('api-version=' + encodeURIComponent(currentApiVersion)); }
```

Comment:
Maybe, it would be better to add a line break in this if statement🤓

Reply:
I prefer to keep it that way since there is no other conditions around this, to keep it simple and minimal.
I may remove braces but to keep it safe for future mistakes (accidental breaks) and to increase readability, I intentionally put braces.

Reply:
Ok, good)

Reply:
I agree with Vlad. The code style should be as consistent as possible. Majority of code arround uses separate line for one-line if statement. I guess we should set up some styling rules for TS similarly to our C# style guide.